rosbag2_transport/include/rosbag2_transport/player.hpp

```diff
@@ -122,6 +134,28 @@ class Player : public rclcpp::Node
   ROSBAG2_TRANSPORT_PUBLIC
   bool play_next();
 
+  // \brief add keyboard callbacks
```

Comment:
I think the auto-generated brief of the function name would have added this much context already ;)